test/CMakeLists.txt

```diff
@@ -26,7 +26,7 @@ macro(ADD_UVW_TEST TEST_NAME TEST_SOURCE)
     add_executable(${TEST_NAME} $<TARGET_OBJECTS:odr> ${TEST_SOURCE})
     target_include_directories(${TEST_NAME} PRIVATE ${UVW_SRC_DIR})
     target_link_libraries(${TEST_NAME} PRIVATE ${COMMON_LINK_LIBS})
-    add_test(NAME uvw-${TEST_NAME} COMMAND ${TEST_NAME})
```

Comment:
Please keep the prefix. It helps when running tests from several parents above.

This is why all libuv tests are called `uv_*`, for example. Otherwise, it's impossible to determine which tests belong where, and further impossible to filter them out.

Reply:
Oh, ok, didn't get the reason. Then let's use the same convention of `libuv`, thus `uvw_`. :+1:

Reply:
Sure :)